client/webserver/site/src/js/order.ts

```diff
@@ -70,8 +70,9 @@ export default class OrderPage extends BasePage {
       this.refreshOnPopupClose = true
     }
     // Do not call cleanTemplates before creating the AccelerateOrderForm
+    OrderUtil.setOptionTemplates(page)
```

Comment:
The `setOptionTemplates` things is kinda a hack anyway. We should really just pass the `page.*Tmpl` elements to the form constructors. But no reason to mix it up now.